src/Symfony/Component/Ldap/LdapClient.php

```diff
@@ -24,13 +24,22 @@
 
     public function __construct($host = null, $port = 389, $version = 3, $useSsl = false, $useStartTls = false, $optReferrals = false, LdapInterface $ldap = null)
     {
+        if ((bool) $useSsl) {
```

Comment:
True. However this was kept to ensure the previous behaviour.

I can remove the cast, if needed.